qubesmanager/settings.py

```diff
@@ -1140,7 +1140,15 @@ def __init_advanced_tab__(self):
 
         if not hasattr(self.vm, "template_for_dispvms"):
             self.dvm_template_checkbox.setEnabled(False)
+
+        if self.dvm_template_checkbox.isChecked():
+            vm_preload_dispvm = int(
+                self.vm.features.get("preload-dispvm-max", 0) or 0
+            )
+        else:
+            vm_preload_dispvm = 0
         self.preload_dispvm.setMinimum(0)
+        self.preload_dispvm.setMaximum(9999)
```

Comment:
That does not seem like a sensible maximum... (Also, max and min can be set in the ui file; easiest way to edit it is with designer-qt)